ads/_config.js

```diff
@@ -350,7 +350,12 @@ export const adConfig = {
   },
 
   pubmine: {
-    prefetch: 'https://s.pubmine.com/showad.js',
+    prefetch: [
+      'https://s.pubmine.com/head.js',
+      'https://s.pubmine.com/showad.js',
```

Comment:
The old script still gets called eventually by the new script. Let me know if it makes sense to not include both in the prefetch.